### PR TITLE
Refactoring: standardize SCM acronym

### PR DESCRIPTION
**Pull request by ncounter**
This PR is only about SCM acronym standardization in the codebase: from `scm|Scm|SCM` occurrences to `SCM` only.

Fixes https://trello.com/c/s5Sr7U2q

### CODE REVIEW

**Comment by ncounter** [comment]
[Convention for names with SCM acronym/abbreviation (S)](https://trello.com/c/s5Sr7U2q/1973-convention-for-names-with-scm-acronym-abbreviation-s)

File: src/api/app/jobs/report_to_scm_job.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # We don't properly capitalize SCM in the class name since CreateJob is doing `CLASS_NAME.to_s.camelize.safe_constantize`
```
Comment (danidoni) [inline src/api/app/jobs/report_to_scm_job.rb:1]: Is this comment still valid, then?

Comment (ncounter) [reply src/api/app/jobs/report_to_scm_job.rb]: At the end of this refactoring, it shouldn't be. I will make sure to drop the comment. :+1:

**Review by krauselukas — CHANGES_REQUESTED** [review]

**Review by danidoni — APPROVED** [review]

**Comment by ncounter** [comment]
Rebased on master, ready to merge.

@danidoni thanks for the review and the approval
@krauselukas if you could give a second review it would be great, thanks

**Review by krauselukas — COMMENTED** [review]
@ncounter the changes look good to me, just the commit message is a bit vague. Maybe you can mention what exactly you refactored.

**Review by krauselukas — APPROVED** [review]

**Pull request merged**